modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,132 @@
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>:8443/ovirt-engine/api`
+** *CA certificate*: Click *Browse* to upload the RHV Manager CA certificate.
+** *Username*: RHV Manager user name
+** *Password*: RHV Manager password
+
+* If you select a saved RHV instance, the wizard connects to the RHV instance using the saved credentials.
+endif::[]
+
+. Click *Check and Save* and wait for the connection to complete.
+
+ifdef::virt-importing-rhv-vm[]
+. Select a cluster and a virtual machine to import.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+. Select a virtual machine or a template to import.
+endif::[]
+
+. Click *Next*.
+
+. In the *Review* screen, review your settings.
+ifdef::virt-importing-rhv-vm[]
++
+Optional: You can select *Start virtual machine on creation*. The RHV virtual machine is powered off before it is imported.
+endif::[]
+
+. If the settings are correct, click *Import*.
++
+A *Successfully created virtual machine* message and a list of resources created for the virtual machine are displayed. The virtual machine appears in *Workloads* -> *Virtual Machines*.
+
+. Optional: Click *Edit* to update the following settings:
+
+* *General*:
+
+** *Name*: If the imported virtual machine name is already being used by another virtual machine in the namespace, update the name.
+** *Description*: Optional
+** *Operating System*
+** *Flavor*
+** *Memory*
+** *CPUs*
+** *Workload Profile*
+
+* *Networking*:
+
+** *Name*
+** *Model*
+** *Network*
+** *Type*
+** *MAC Address*
++
+Optional: Click *Add Network Interface* to add an interface.
+
+* *Storage*:
+
+** *Name*
+** *Source*: For example, *Import Disk*
+** *Size*
+** *Interface*
+ifdef::virt-importing-rhv-vm[]
+** *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default `nfs-storage` storage class to create the virtual machine.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+** *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class to create the virtual machine.
+endif::[]
+** *Advanced* > *Volume Mode*
+** *Advanced* > *Access Mode*
++
+Optional: Click *Add Disk* to add a disk to the virtual machine.
+
+ifdef::virt-importing-vmware-vm[]
+* *Advanced* > *Cloud-init*:
+
+** *Form*: Enter the *Hostname* and *Authenticated SSH Keys*.
+** *Custom script*: Enter the `cloud-init` script in the text field.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+* *Advanced* > *Cloud-init*: `Not Enabled`
++
+`cloud-init` is not imported with the RHV source virtual machine. You must recreate the `cloud-init` settings after import.
+endif::[]
+* *Advanced* > *Virtual Hardware*: You can attach a virtual CD-ROM to the imported virtual machine.
+
+. If the settings are correct, click *Review and Import*.
++
+A *Successfully created virtual machine* message and a list of resources created for the virtual machine are displayed. The powered off virtual machine appears in *Workloads* -> *Virtual Machines*.
```

Comment:
This doc describe how to import from the UI. We also support CLI for importing from RHV. Do we want to cover it as well?

Reply:
Not unless there's a very good reason for including the CLI method.

Reply:
@dankenigsberg any opinions on this?

Reply:
I'm an advocate of documenting CLI/API. I would appreciate it if we can add an annotated example of how to initiate and track an import from the command line. This would help sophisticated users (e.g the mass-migration team) to build upon the single VM story.

@pkliczewski would you provide a paragraph for @apinnick to consider? Something like
"""
You may also trigger an import from the command line, by editing the following `import.yaml` and applying it with `oc apply -f import.yaml`. Use `oc get ....` to track progress.
"""

Reply:
We already have this described in the [readme](https://github.com/kubevirt/vm-import-operator#import-virtual-machine-from-ovirt).  @apinnick please let me know if this is something you could use.

Reply:
@pkliczewski @dankenigsberg 

If you want the CLI documented, you will need to move the "operator.yml" to some place more secure, like registry.redhat.com. Downstream docs is not supposed to link to Github repos.

Reply:
> I'm an advocate of documenting CLI/API. I would appreciate it if we can add an annotated example of how to initiate and track an import from the command line. This would help sophisticated users (e.g the mass-migration team) to build upon the single VM story.

I agree that tracking an import from the command line is a good thing. However, this particular section was never intended for mass RHV VM migration. That's IMS 2.0, which will have its own UI. This PR is only for a single VM.

Reply:
I'm suggesting to copy a short example into here. OpenShift docs sometimes have embedded annotated CRs, which I like (e.g in https://docs.openshift.com/container-platform/4.3/networking/multiple_networks/configuring-host-device.html#nw-multus-host-device-object_configuring-host-device )

I find it useful to track even a single vm import from the command line.

I think that adding these docs would help future customers and developers alike.

Reply:
Agree that tracking the VM import from the command line is useful. 

I just wanted to make it clear that we are not encouraging people to build their own operators for mass migration.

Reply:
@dankenigsberg @pkliczewski 

Are you saying that the instructions should be something like this?

1. Create import.yml file, using this [example](https://github.com/kubevirt/vm-import-operator/blob/master/examples/ovirt_virtualmachineimport_example_cr.yaml).
2. Trigger the import: `$ oc apply -f import.yml`
3. Follow the import progress with `$ oc get vmimports example-virtualmachineimport -n default`

I'm not sure. I would probably have to add instructions for filling in the parameters and a troubleshooting section. I don't want to include a procedure that might not work, and I'm not sure I have all the info.

Reply:
The instructions are correct, you can rely on a simpler [example](https://github.com/kubevirt/vm-import-operator/blob/master/examples/ovirt/vmimport-internal-mapping.yaml)

The steps you've mentioned are correct.